config.json

```diff
@@ -1,836 +1,793 @@
 {
-  "language": "Haskell",
   "active": true,
-  "solution_pattern": "example.*[.]hs",
-  "ignore_pattern": "example",
   "exercises": [
     {
-      "uuid": "c97d9dda-c7ee-4595-b447-693559d03ba5",
-      "slug": "hello-world",
       "core": false,
-      "unlocked_by": null,
       "difficulty": 1,
-      "topics": [
-      ]
+      "slug": "hello-world",
```

Comment:
slug seems important. it seems like it should come first or second (behind uuid)

Reply:
slug first please.